packages/mediacenter/JsonSchemaBuilder/package.mk

```diff
@@ -8,10 +8,9 @@ PKG_LICENSE="GPL"
 PKG_SITE="http://www.kodi.tv"
 PKG_DEPENDS_HOST="toolchain:host"
 PKG_DEPENDS_UNPACK="kodi"
-PKG_NEED_UNPACK="$(get_pkg_directory $MEDIACENTER)"
 PKG_LONGDESC="kodi-platform:"
 
-PKG_CMAKE_SCRIPT="$(get_build_dir $MEDIACENTER)/tools/depends/native/JsonSchemaBuilder/CMakeLists.txt"
+PKG_CMAKE_SCRIPT="$(get_build_dir kodi)/tools/depends/native/JsonSchemaBuilder/CMakeLists.txt"
 
```

Comment:
Please use `$MEDIACENTER` instead of `kodi`, and `PKG_DEPENDS_UNPACK="kodi"` should be changed to `PKG_DEPENDS_UNPACK="$MEDIACENTER"`

Reply:
done